packages/utils/tokens/src/layout-tokens.ts

```diff
@@ -6,6 +6,7 @@ import type { OperationSet } from './token.types';
 import { tokenBuilder } from './tokenBuilder';
 
 export interface LayoutTokens {
+  width?: ViewStyle['width'];
```

Comment:
should we add height as well? Also, did we confirm that no other set of tokens uses width/height?

Reply:
OK I don't see any other token sets that use width/height. But I would prefer we _also_ add height.